### PR TITLE
ESSL / Khronos updates

### PR DESCRIPTION
**Pull request by bernardkwok**
Update #1184 

Backmerges from main related to ESSL, Khronos work
- multiple mesh support in view (Khronos)
- combined GLSL/ESSL module
- validation via glslang

### CODE REVIEW

File: source/JsMaterialX/CMakeLists.txt [hunk 1]
```diff
@@ -111,7 +111,7 @@ target_link_libraries(JsMaterialXGenShader
     PUBLIC MaterialXCore
     PUBLIC MaterialXFormat
     PUBLIC MaterialXGenShader
-    PUBLIC MaterialXGenEssl
+    PUBLIC MaterialXGenGlsl
```
Comment (ashwinbhat) [inline source/JsMaterialX/CMakeLists.txt:114]: Don't understand this

File: source/JsMaterialX/JsMaterialXGenEssl/JsEsslShaderGenerator.cpp [hunk 2]
```diff
@@ -3,7 +3,7 @@
 // All rights reserved.  See LICENSE.txt for license.
 //
 
-#include <MaterialXGenEssl/EsslShaderGenerator.h>
+#include <MaterialXGenGlsl/EsslShaderGenerator.h>
```
Comment (ashwinbhat) [inline source/JsMaterialX/JsMaterialXGenEssl/JsEsslShaderGenerator.cpp:6]: Essl is moved to MaterialXGenGlsl?

**Review by ashwinbhat — APPROVED** [review]
LGTM

File: source/JsMaterialX/CMakeLists.txt (hunk 1 above)
Comment (bernardkwok) [reply source/JsMaterialX/CMakeLists.txt:114]: There is no new MaterialXGenEssl. Just reuse the existing library.

File: source/JsMaterialX/JsMaterialXGenEssl/JsEsslShaderGenerator.cpp (hunk 2 above)
Comment (bernardkwok) [reply source/JsMaterialX/JsMaterialXGenEssl/JsEsslShaderGenerator.cpp:6]: Yes.

**Pull request merged**